### ISSUE TITLE
Pick starting point [Feature request]

### ISSUE THREAD

**Issue by troyml42** [matejker/everystreet#4]
It would be great if the starting point of the circuit could be selected such that a location with good parking for instance might be selected, or a person's home.

**Comment by kevinp2** [comment]
I agree that this would be a good feature.  Note that it would change the route though, possibly making it longer.
